eidpauth/src/main/java/uk/ac/ucl/eidp/auth/jaas/EidpLoginModule.java

```diff
@@ -45,16 +46,23 @@ private void setUserService() {
         Context context = new InitialContext();
         beanManager = (BeanManager) context.lookup("java:comp/BeanManager");
         Bean<?> bean = beanManager.getBeans(UserController.class).iterator().next();
-        CreationalContext cc = beanManager.createCreationalContext(bean);
-        userService = (UserController) beanManager.getReference(bean, UserController.class, cc);
-      } catch (NamingException e) {
-        logger.log(Level.SEVERE, "Cannot call UserController bean", e);
+        CreationalContext creationalCtx = beanManager.createCreationalContext(bean);
+        userService = (UserController) beanManager.getReference(
+          bean, UserController.class, creationalCtx
+        );
+      } catch (NamingException namingX) {
+        logger.log(Level.SEVERE, "Cannot call UserController bean", namingX);
       }
     }
   }
 
   @Override
-  public void initialize(Subject subject, CallbackHandler callbackHandler, Map<String, ?> sharedState, Map<String, ?> options) {
+  public void initialize(
+      Subject subject, 
+      CallbackHandler callbackHandler, 
+      Map<String, ?> sharedState, 
+      Map<String, ?> options
+    ) {
```

Comment:
\[Checkstyle\] WARNING: 'method def rparen' have incorrect indentation level 4, expected level should be 2\.